### PR TITLE
deps: @metamask/utils@^6.2.0->^8.1.0

### PR DESCRIPTION
**Pull request by legobeat**
## Explanation

[Changelog](https://github.com/MetaMask/utils/blob/main/CHANGELOG.md)

### `@metamask/acccounts-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/address-book-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/approval-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/assets-controllers`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/base-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/controller-utils`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/ens-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/gas-fee-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/keyring-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/message-manager`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/name-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/network-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/notification-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/permission-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/polling-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/signature-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

### `@metamask/transaction-controller`
  - **deps**: Update `@metamask/utils` from `^6.2.0`  to `^8.1.0`

## Checklist

- [ ] I've updated the test suite for new or updated code as appropriate
- [ ] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
- [ ] I've highlighted breaking changes using the "BREAKING" category above as appropriate

### CODE REVIEW

File: packages/permission-controller/src/rpc-methods/getPermissions.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { MethodNames } from '../utils';
 
 export const getPermissionsHandler: PermittedHandlerExport<
   GetPermissionsHooks,
-  undefined,
+  any,
```
Comment (MajorLift) [inline packages/permission-controller/src/rpc-methods/getPermissions.ts:11]:
```suggestion
[],
```

Using `[]` as the empty value here also resolves the compiler errors. It's a little hacky, but I think it should be safer than using `any`.

**Comment by MajorLift** [comment]
Otherwise LGTM!

**Review by MajorLift — DISMISSED** [review]
Otherwise LGTM! 

Looks like there are a number of open PRs depending on this update.
https://github.com/MetaMask/core/pull/1724
https://github.com/MetaMask/core/pull/1653
https://github.com/MetaMask/eth-json-rpc-provider/pull/28

Closes https://github.com/MetaMask/core/pull/1729

**Review by MajorLift — APPROVED** [review]

**Review by Mrtenz — DISMISSED** [review]

**Review by MajorLift — DISMISSED** [review]

**Review by montelaidev — APPROVED** [review]
Lgtm

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**